### PR TITLE
Update recert presubmits for OCP 4.14

### PR DESCRIPTION
**Pull request by mresvanis**
This PR:

- moves [recert](https://github.com/rh-ecosystem-edge/recert) `main` branch presubmits to OpenShift 4.14
  The changes required for recert in OpenShift 4.14 are due to the changes of [multus](https://github.com/openshift/multus-cni/commit/30558b5e68d3fd316fefa35fa7d258b91773f8a0) and [OVN](https://github.com/openshift/ovn-kubernetes/commit/7dc48048a346f175772fe5c2839b083f7a20c597) regarding per-node certificates.
- enhances the recert script to:
  - reorder script functions to reflect their call order
  - fetch the etcd image from the running cluster, instead of relying to
    a hardcoded release image
  - wait for the temp etcd to be available, instead of `sleep`-ing for
    10s

### CODE REVIEW

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-parallel
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-serial

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-parallel

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-parallel

**Comment by mresvanis** [comment]
@omertuc this PR is ready for review :pray:

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-parallel pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-serial pull-ci-rh-ecosystem-edge-recert-main-cargo-check pull-ci-rh-ecosystem-edge-recert-main-cargo-clippy pull-ci-rh-ecosystem-edge-recert-main-images

File: ci-operator/step-registry/single-node/recert/single-node-recert-commands.sh [hunk 1]
```diff
@@ -32,9 +50,17 @@ function stop_containers {
   systemctl stop crio
 }
 
+function wait_for_recert_etcd {
+  echo "Waiting for recert etcd to be available..."
+  until curl -L http://localhost:2379/version &> /dev/null
```
Comment (omertuc) [inline ci-operator/step-registry/single-node/recert/single-node-recert-commands.sh:55]:
```suggestion
  until curl -s http://localhost:2379/health | jq -e '.health == "true"' &> /dev/null
```

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-parallel pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-serial pull-ci-rh-ecosystem-edge-recert-main-cargo-check pull-ci-rh-ecosystem-edge-recert-main-cargo-clippy pull-ci-rh-ecosystem-edge-recert-main-images

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-parallel pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-serial pull-ci-rh-ecosystem-edge-recert-main-cargo-check pull-ci-rh-ecosystem-edge-recert-main-cargo-clippy pull-ci-rh-ecosystem-edge-recert-main-images

**Pull request merged**